scripts/qiita-private-launcher

```diff
@@ -0,0 +1,53 @@
+#!/usr/bin/env python
```

Comment:
The private launcher is a notion that I've introduced for first time in this PR. The scripts `qiita-private-launcher` and `qiita-plugin-launcher` are two scripts tailored to be optimized by the system administrators so they decide which is the best way to execute parallel jobs. In the code added to the repository, these scripts are optimized to work in a laptop environment. In our servers, however, these scripts have been modified so instead of doing a direct bash execution, they execute a qsub submitting the job to the queue.

The private launcher is a script that used to submit jobs that are performing internal Qiita processes. The only functionality in which they're used at this moment is to complete the jobs. When a job is completed, artifacts are created (which involves file movement) so it is important to perform this in a different job rather than in the webserver itself. The main reason to use a different script instead of using qiita-plugins-launcher is that the internal Qiita jobs are more controlled and in general way lighter, so having a different script allows optimization of submission parameters of this jobs.

Hope this makes a bit more sense! And thanks for asking, this forces me to try to put in words the concepts that I have on my head.